src/main/java/org/terasology/gookeeper/Constants.java

```diff
@@ -2,12 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.terasology.gookeeper;
 
-@SuppressWarnings("checkstyle:HideUtilityClassConstructor")
-public class Constants {
+public final class Constants {
     public static final String VISITOR_SPAWN_DELAY_EVENT_ID = "VISITOR_SPAWN_DELAY";
     public static final String DESTROY_ARROW_EVENT_ID = "DESTROY_ARROW";
     public static final String HEALTH_DECREASE_EVENT_ID = "DECREASE_HEALTH_TICK";
     public static final String GOOEY_DEATH_EVENT_ID = "DESTROY_GOOEY_ENTITY";
     public static final String SPAWN_GOOEY_EGG_EVENT_ID = "SPAWN_GOOEY_EGG";
     public static final String HATCH_EGG_EVENT_ID = "HATCH_GOOEY_EGG";
+    private Constants() {
```

Comment:
Should be preceded by empty line
```suggestion

    private Constants() {
```